app/views/alert.html

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+	<div class="col-sm-8 col-sm-offset-2" id="login">
+		<div class="alert alert-{{alert.type}} alert-dismissible" role="alert" ng-repeat="alert in alerts" type="{{alert.type}}" id="alert-{{$index}}">
```

Comment:
`alert-dismissable` in combination with the button on the next line - now the alert is properly dismissable!